suite-native/navigation/src/components/NavigationContainerWithAnalytics.tsx

```diff
@@ -1,4 +1,5 @@
 import { useMemo, useRef, ReactNode } from 'react';
+import * as Sentry from '@sentry/react-native';
```

Comment:
nitpick: I would prefer to import only `addBreadcrumb` instead of whole Sentry, but don't know if it makes any difference.

Reply:
following the usage from elsewhere here

Reply:
I understand, and it's a nitpick, so feel free to ignore it if you find it irrelevant